crates/paralegal-policy/src/context.rs

```diff
@@ -343,17 +477,14 @@ impl Context {
     /// Return an example pair for a flow from an source from `from` to a sink
     /// in `to` if any exist.
     pub fn any_flows<'a>(
-        &self,
-        ctrl_id: Option<ControllerId>,
-        from: &[&'a DataSource],
-        to: &[&'a CallSiteOrDataSink],
+        &'a self,
+        from: &[&'a CtrlNode],
```

Comment:
So since `CtrlNode` is now small (e.g. implements `Copy`) I think we should use `CtrlNode` instead of `&CtrlNode` everywhere.

Reply:
Done